build/serve.js

```diff
@@ -20,7 +20,8 @@ import browserSyncSpa from 'browser-sync-spa';
 import child from 'child_process';
 import gulp from 'gulp';
 import path from 'path';
-import proxyMiddleware from 'proxy-middleware';
+// import proxyMiddleware from 'proxy-middleware';
+import proxyMiddleware from 'http-proxy-middleware';
```

Comment:
Changed to using http-proxy-middleware because it supports websockets but BrowserSync itself doesn't seem to work with websockets.